Simplenote/src/main/res/layout/dialog_wordpress_post.xml

```diff
@@ -1,158 +1,160 @@
+            </CheckBox>
 
             <Space
                 android:layout_width="0dp"
                 android:layout_height="0dp"
-                android:layout_weight="1" />
+                android:layout_weight="1" >
+            </Space>
+
         </LinearLayout>
+
     </LinearLayout>
 
     <LinearLayout
         android:id="@+id/wp_dialog_section_posting"
-        android:layout_width="match_parent"
+        android:layout_gravity="center_vertical"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
-        android:paddingLeft="24dp"
-        android:paddingRight="24dp"
-        android:paddingTop="@dimen/padding_large"
+        android:layout_width="match_parent"
+        android:orientation="horizontal"
         android:paddingBottom="@dimen/padding_large"
-        android:orientation="vertical"
-        android:visibility="gone">
+        android:paddingEnd="@dimen/padding_extra_extra_large"
+        android:paddingLeft="@dimen/padding_extra_extra_large"
+        android:paddingRight="@dimen/padding_extra_extra_large"
+        android:paddingStart="@dimen/padding_extra_extra_large"
+        android:paddingTop="@dimen/padding_large"
+        android:visibility="gone" >
+
+        <ProgressBar
+            android:indeterminate="true"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/padding_extra_extra_large"
+            android:layout_marginRight="@dimen/padding_extra_extra_large"
+            android:layout_width="wrap_content" >
+        </ProgressBar>
 
         <TextView
-            android:layout_width="match_parent"
+            android:ellipsize="end"
+            android:layout_gravity="center_vertical"
             android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/padding_medium"
-            android:layout_marginBottom="@dimen/padding_medium"
-            android:gravity="center_horizontal"
+            android:layout_width="fill_parent"
+            android:maxLines="1"
             android:text="@string/uploading_post"
-            android:textColor="?attr/noteTitleColor" />
+            android:textColor="?attr/noteTitleColor"
+            tools:ignore="RtlSymmetry" >
+        </TextView>
 
-        <ProgressBar
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/padding_large"
-            android:gravity="center_horizontal"
-            android:indeterminate="true" />
     </LinearLayout>
 
     <RelativeLayout
         android:id="@+id/wp_dialog_section_success"
-        android:layout_width="match_parent"
+        android:layout_gravity="center_horizontal"
         android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:paddingBottom="@dimen/padding_large"
         android:paddingLeft="24dp"
         android:paddingRight="24dp"
```

Comment:
These could be `padding_extra_extra_large`?

Reply:
Yep.  I updated the padding values in 37cd714.